src/scss/_mixins.scss

```diff
@@ -0,0 +1,143 @@
+/// Apply styles at a given layout size
+/// Wrapper for the Sass MQ mq() mixin
+///
+/// @link https://git.io/sass-mq Sass MQ documentation
+///
+/// @example
+///  // Turn the color of an element red at medium layout size and up
+///  @include oLayoutBreakPoint(M) {
+///  	element {
+///  		color: red;
+///  	}
+///  }
+///  // Turn the color of an element blue until medium layout
+///  element {
+///  	@include oLayoutBreakPoint($until: M) {
+///  		color: blue;
+///  	}
+///  }
+///  // Turn the color of an element green from small layout until medium layout
+///  element {
+///  	@include oLayoutBreakPoint($from: S, $until: M) {
+///  		color: green;
+///  	}
+///  }
+///
+/// @param {String} from - one of $o-layout-breakpoints
+/// @param {String} until - one of $o-layout-breakpoints
+
+@mixin oLayoutBreakPoint($from: false, $until: false) {
+	@include mq(
+		$from: $from,
+		$until: $until,
+		$breakpoints: $o-layout-breakpoints
+	) {
+		@content;
+	}
+}
+
+/// Base grid
+@mixin _oLayoutGridBase () {
+	min-width: 100%;
+	margin: 0;
+	position: relative;
+	box-sizing: border-box;
+
+	display: grid;
+	grid-auto-rows: auto;
+	grid-template-columns: 100%;
+	grid-gap: $_o-layout-gutter;
+	grid-template-areas:
+		"header"
+		"sidebar"
+		"main"
+		"footer";
+}
+
+/// Base styling for the layout content
+@mixin _oLayoutBody () {
+	@include oTypographySans(1);
+
+	a {
```

Comment:
Is this duplicating what `@include oTypographyWrapper($style: product);` should do?